src/Components/RelatedProducts/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
a basic product card component with placeholders for all of the relevant pieces of information required by the business docs